pkgs/development/python-modules/minc2-simple/default.nix

```diff
@@ -0,0 +1,23 @@
+{ buildPythonPackage, fetchFromGitHub, stdenv, cmake,
+  hdf5, libminc, minc2_simple, minc_tools, mni_autoreg,
+  cffi, six, numpy, scipy, pytest }:
+
+buildPythonPackage {
+  pname   = "minc2-simple";
+  version = minc2_simple.version;
+
+  src = minc2_simple.src;
+
+  checkInputs = [ pytest minc_tools mni_autoreg ];
+  buildInputs = [ cffi six numpy scipy hdf5 libminc minc2_simple ];
+
+  sourceRoot = "source/python";
```

Comment:
i would move sourceRoot above the inputs, as that follows the order of how the package gets built :)